src/main/java/net/quickwrite/fluent4j/ast/FluentAttribute.java

```diff
@@ -0,0 +1,48 @@
+package net.quickwrite.fluent4j.ast;
+
+import net.quickwrite.fluent4j.util.StringSlice;
+
+
+/**
+ * A FluentAttribute defines a subpart of a FluentMessage that holds more
+ * context data.<br>
+ *
+ * <p>
+ * <pre>
+ * login-input = Predefined value
+ *     .placeholder = email@example.com
+ *     .aria-label = Login input value
+ *     .title = Type your login email
+ * </pre>
+ * <p>
+ * In this example the {@code login-input} has three different attributes:
+ * A{@code placeholder} attribute, {@code aria-label} attribute, and a {@code title} attribute.
```

Comment:
There is a space missing.

```suggestion
 * A {@code placeholder} attribute, {@code aria-label} attribute, and a {@code title} attribute.
```